src/main/java/com/maarten551/tictactoe_backend/controller/LobbyController.java

```diff
@@ -0,0 +1,147 @@
+package com.maarten551.tictactoe_backend.controller;
+
+import com.maarten551.tictactoe_backend.dto.PlayerExecutionMove;
+import com.maarten551.tictactoe_backend.dto.SelectedLobbyOverview;
+import com.maarten551.tictactoe_backend.exception.*;
+import com.maarten551.tictactoe_backend.logic.GameSessionContainer;
+import com.maarten551.tictactoe_backend.logic.LobbyContainer;
+import com.maarten551.tictactoe_backend.logic.PlayerContainer;
+import com.maarten551.tictactoe_backend.model.Lobby;
+import com.maarten551.tictactoe_backend.model.Player;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.messaging.handler.annotation.*;
+import org.springframework.messaging.simp.SimpMessageHeaderAccessor;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.messaging.simp.annotation.SendToUser;
+import org.springframework.messaging.simp.annotation.SubscribeMapping;
+import org.springframework.stereotype.Controller;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Controller
+public class LobbyController {
+	public static final String LOBBY_OVERVIEW_ENDPOINT = "/lobbies";
+    public static final String LOBBY_JOINED_ENDPOINT = "/lobby/{lobbyId}";
+
+    private final LobbyContainer lobbyContainer;
+    private final PlayerContainer playerContainer;
+    private final GameSessionContainer gameSessionContainer;
+    private final SimpMessagingTemplate template;
+
+	@Autowired
+    public LobbyController(LobbyContainer lobbyContainer, PlayerContainer playerContainer, GameSessionContainer gameSessionContainer, SimpMessagingTemplate template) {
+		this.lobbyContainer = lobbyContainer;
+		this.playerContainer = playerContainer;
+        this.gameSessionContainer = gameSessionContainer;
+        this.template = template;
+	}
+
+	@SubscribeMapping(LOBBY_OVERVIEW_ENDPOINT)
+	public List<Lobby> onLobbyOverview() {
```

Comment:
Function is never used? Also, better naming might be possible.

Reply:
This function is used & registered by the annotation.

When a client subscribes on this channel (the value inside `LOBBY_OVERVIEW_ENDPOINT`), all the current lobbies are returned to the client.

In this case, when the client wants to be notified about all the lobbies that are waiting for new players, all the current open lobbies are returned.

Reply:
je ne comprends pas de rien

Reply:
To be fair, I didn't understand your french.